css/style.css

```diff
@@ -3146,4 +3146,13 @@ a:hover, a:focus, a:active {
 
 .dropdown2:hover .dropdown-content {
   display: block;
-}
\ No newline at end of file
+}
+
+.contributing-button {
+  width:25%;
+  font-size: 20px;
+  align-items: center;
+  position:relative;
+  left:38%;
```

Comment:
Instead of `left:38%`, use `margin: auto`.
This will automatically center the button.